lib/rex/parser/arguments.rb

```diff
@@ -70,26 +84,39 @@ def parse(args, &_block)
       def usage
         txt = ["\nOPTIONS:\n"]
 
-        fmt.sort.each do |entry|
+        fmt.sort_by { |option| [option.first].flatten.first.downcase }.each do |entry|
           fmtspec, val = entry
           opt = val[0] ? " <opt>  " : "        "
-          txt << "    #{fmtspec.ljust(longest.length)}#{opt}#{val[1]}"
+
+          # Get all arguments for a command
+          output = [fmtspec].map { |value| [value].flatten.join(', ') }.first
```

Comment:
Following this suggestion to normalize the data on entry https://github.com/rapid7/metasploit-framework/pull/15944/files#r768637437

Would allow you to remove all the `[value].flatten` handling in the rest of the file :+1: